resources/client/components/AudioRecorder/useAudioRecorder.ts

```diff
@@ -0,0 +1,154 @@
+import { ref, onUnmounted } from "vue";
+
+function getSupportedMimeType() {
+  const types = [
+    "audio/mp4", // prefer mp4 for safari support
+    "audio/webm",
+    "audio/ogg;codecs=opus",
+    "audio/wav",
```

Comment:
do we need to allow wav support? even with a 15s limit, that's gonna eat a decent chunk of storage